src/main/java/fr/moribus/imageonmap/commands/IoMCommand.java

```diff
@@ -53,6 +56,23 @@
 public abstract class IoMCommand extends Command {
 
 
+    protected boolean checkHostingSite(URL url) {
+        PluginLogger.info("allow list " + PluginConfiguration.ALLOWLIST_HOSTINGSITE.get());
+        String urlsString = PluginConfiguration.ALLOWLIST_HOSTINGSITE.get();
+        if (urlsString.trim().equals("")) {
```

Comment:
`.isEmpty()`?

Reply:
Yeah I haven't seen that in the end I was testing if it was empty, thanks.